docs/source/sci_api_tutorial/assign_mutant.rst

```diff
@@ -0,0 +1,355 @@
+        
+        test_A = "test_A.pdb"
+        test_A_stru = PDBParser.get_structure(test_A)
+        test_mutation_pattern_A = "F392G"
+        mutants_A = mapi.assign_mutant(test_A_stru, test_mutation_pattern_A)
+        print(mutants_A)
+        #[[('PHE','GLY','A',392)]]
+
+* Select three seperate signle-point mutation sites in a monomer protein, e.g. mutate P329 to G392, R378 to E378, D211 to T211.
+    
+    .. code:: python
+        
+        test_A = "test_A.pdb"
+        test_A_stru = PDBParser.get_structure(test_A)
+        test_mutation_pattern_A = "F392G,R378E,D211T"
+        mutants_A = mapi.assign_mutant(test_A_stru, test_mutation_pattern_A)
+        print(mutants_A)
+        #[[('PHE','GLY','A',392)], [('ARG','GLU','A',378)], [('ASP','THR','A',211)]]
+
+* Select a three-point mutation in a monomer protein, e.g. mutate P329 to G392, R378 to E378, D211 to T211 simultaneously.
+    
+    .. code:: python
+        
+        test_A = "test_A.pdb"
+        test_A_stru = PDBParser.get_structure(test_A)
+        test_mutation_pattern_A = "{F392G,R378E,D211T}"
+        mutants_A = mapi.assign_mutant(test_A_stru, test_mutation_pattern_A)
+        print(mutants_A)
+        #[[('PHE','GLY','A',392), ('ARG','GLU','A',378), ('ASP','THR','A',211)]]
+
+* Generate two mutant spaces, one is three-point mutation and another is two-point mutaion in a monomer protein, e.g. mutate P329 to G392, R378 to E378, D211 to T211 in one mutant; N363 to E363 and M71 to L71 in another mutant.
+    
+    .. code:: python
+        
+        test_A = "test_A.pdb"
+        test_A_stru = PDBParser.get_structure(test_A)
+        test_mutation_pattern_A = "{F392G,R378E,D211T}","{N363E,M71L}"
+        mutants_A = mapi.assign_mutant(test_A_stru, test_mutation_pattern_A)
+        print(mutants_A)
+        #[[('ASP','THR','A',211), ('PHE','GLY','A',392), ('ARG','GLU','A',378)], 
+        # [('ASN','GLU','A',363), ('MET','LEU','A',71)]]
+
+* Select three two signle-point mutation sites in a homologous two-chain protein, and the two mutation locates on different chains, e.g. mutate L383 to H383 on chain A, N363 to E363 on chain B.
+    
+    .. code:: python
+        
+        test_A_B = "test_A_B.pdb"
+        test_A_B_stru = PDBParser.get_structure(test_A_B)
+        test_mutation_pattern_A_B = "LA383H,NB363E"
+        mutation_pattern_A_B = mapi.assign_mutant(test_A_B_stru, 
+                                                  test_mutation_pattern_A_B, 
+                                                  chain_sync_list=[("A", "B")], 
+                                                  chain_index_mapper={"A": 0, "B": 0})
+        print(mutation_pattern_A_B)
+        #[[('LEU','HIS','A',383), ('LEU','HIS','B',383)], [('ASN','GLU','B',363), ('ASN','GLU','A',363)]]
+
+* Select one specific mutation site (e.g. M71 to L71), combined with randomly generate 5 two-point mutations around residue 289 within a 4 Å radius, excluding residue 36, to larger amino acids.
```

Comment:
Have only a simplest and a hardest example highlighted and have all the examples here folded.

Reply:
In the non-folding section, John also suggest to have a chemistry-inspired example (e.g.: create a hole in the enzyme)

Reply:
"most basic usage + chemistry-inspired examples"